hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -35,11 +37,26 @@
 from hyp3_gamma.rtc.coregistration import CoregistrationError, check_coregistration
 from hyp3_gamma.util import set_pixel_as_point, unzip_granule
 
+
 log = logging.getLogger()
 gdal.UseExceptions()
 ogr.UseExceptions()
 
 
+def createPowerDB(fi, nodata=None):
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), function names should be lower case with underscores.  Admittedly not all of the code in hyp3-gamma follows that standard (yet).  I'd suggest maybe:

```suggestion
def create_decibel_tif(fi, nodata=None):
```